docs/README.md

```diff
@@ -9,6 +9,7 @@ In both cases, the generated output can be found in `generated/docs`.
 If you have an [existing Envoy development environment](https://github.com/envoyproxy/envoy/tree/main/bazel#quick-start-bazel-build-for-developers), you should have the necessary dependencies and requirements and be able to build the documentation directly.
 
 ```bash
+export ENVOY_SRCDIR=$(pwd)
```

Comment:
how about just setting it in build.sh ?

Reply:
Make sense but afraid about CI.

Reply:
i added it in a very wip pr im working on (https://github.com/envoyproxy/envoy/pull/15229/files#diff-bcdbd66bdb41241ada207d8561edeabbcc8e8df5a2be4adaf802f239975768c3R43)

i think its ok - lets try it and see

Reply:
I will update in build.sh